src/handler.lisp

```diff
@@ -25,7 +26,12 @@
              (apply (intern #.(string '#:run) handler-package)
                     app
                     :allow-other-keys t
-                    args)))
+                    (append
+                      (and address-specified-p
```

Comment:
This is a matter of taste, but I prefer to use `when` in such cases as a more semantic form.

Reply:
Thanks, I prefer this style :)